### DISCUSSION TITLE
select creatable

### DISCUSSION THREAD

**Discussion by jaallaall** [mantinedev/mantine#7002] [Q&A]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.12.2

### What package has an issue?

@mantine/core

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

None

### Describe the bug

Hello
Why is there no option to add a new item in the Select component?
The creatable option is not available

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
You can find an example of creatable select here – https://mantine.dev/combobox/?e=SelectCreatable

**Reply by jaallaall** [reply]
But I meant for the select component

**Reply by rtivital** [reply]
Yes, the example shows how to create a select component with creatable feature
